components/python/scour/Makefile

```diff
@@ -0,0 +1,54 @@
+# CDDL HEADER START
+#
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Geoff Weiss
+#
+
+BUILD_STYLE=setup.py
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=         scour
+COMPONENT_VERSION=      0.38.2
+COMPONENT_FMRI=         library/python/scour
+COMPONENT_CLASSIFICATION=Development/Python
+COMPONENT_SUMMARY=      An SVG optimizer/cleaner written in Python that reduces the size of scalable vector graphics by optimizing structure and removing unnecessary data.
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= \
+    sha256:6881ec26660c130c5ecd996ac6f6b03939dd574198f50773f2508b81a68e0daf
+COMPONENT_ARCHIVE_URL=  $(call pypi_url)
+COMPONENT_PROJECT_URL=  https://pypi.org/project/scour/
+COMPONENT_LICENSE=      APACHE
+COMPONENT_LICENSE_FILE= scour.license
+
+include $(WS_MAKE_RULES)/common.mk
```

Comment:
If tests don't work for us then disable them by adding
TEST_TARGET= $(NO_TESTS)
here with a comment why tests have been disabled.

Reply:
Thank you.  I used the terminator python module to determine where these lines should be placed.  I ran gmake sample-manifest, gmake test, and gmake publish with this change.  I am now getting the expected behavior.